### ISSUE TITLE
Name conflicts for mamba

### ISSUE THREAD

**Issue by jbednar** [state-spaces/mamba#3]
This looks like a cool project, but the name seems problematic because of the pre-existing package manager [Mamba](https://github.com/mamba-org/mamba).  The Mamba SSM installation instructions currently suggest installing with pip, but Mamba SSM could very soon be available as a conda package, and then you'll have instructions like `mamba install mamba-ssm`, which seems very, very  confusing!

**Comment by mcg1969** [comment]
Indeed, `mamba` is reserved on PyPi for the other package: https://pypi.org/project/mamba/

**Comment by trap20** [comment]
There also is a Llama2 fine tune called mamba: https://huggingface.co/CobraMamba

**Comment by mcg1969** [comment]
To be fair! The official Python name of this package is `mamba_ssm` https://pypi.org/project/mamba-ssm/

So I think we're good from a pip and conda perspective!

**Comment by jbednar** [comment]
If it's consistently called "Mamba SSM", there's no problem!  But the name of this repo, its readme, and the announcement all just use Mamba, so I'd bet lots of people will try to "pip install mamba".

**Comment by josejg** [comment]
Seems like Gemini can't tell the two projects apart (see the generated github link)

![image](https://github.com/state-spaces/mamba/assets/26491792/871a11c5-775b-43b4-b069-7f331577a032)


From: https://twitter.com/mvpatel2000/status/1732496433318592842
